google_api_availability_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0+1
+
+* Declares `dartPluginClass` in pubspec declaration.
```

Comment:
I am not sure why these files are in the PR, they are already in the main branch right? Is your branch up to date with main?

Reply:
Turns out it was not. I have reread https://github.com/flutter/flutter/wiki/Tree-hygiene to learn how to do it and fixed it.